website/docs/components/form/input-group/partials/specifications/anatomy.md

```diff
@@ -0,0 +1,26 @@
+## Anatomy
+
+### Base component
+
+![Input Group Base Anatomy](/assets/components/form/input-group/input-group-base-anatomy.png =800x*)
```

Comment:
This image appears to be stretched and grainy. I think the size declaration is probably a little off.